common/src/main/scala/com/gu/recipeasy/models/RecipePreparation.scala

```diff
@@ -0,0 +1,11 @@
+package com.gu.recipeasy.models
+
+import java.time.OffsetDateTime
+
+import com.gu.recipeasy.db.DB
+
+object RecipePreparation {
+  def selectRecipes(db: DB): List[Recipe] = {
+    db.getRecipes().filter(recipe => ((recipe.ingredientsLists.lists.size >= 5) && (recipe.status == New)))
```

Comment:
Yes. Sorry I should have made that clear. It's just jump-starting the `New` to `Ready` migration